src/aks-preview/azext_aks_preview/_consts.py

```diff
@@ -284,3 +284,7 @@
 # Node Provisioning Mode Consts
 CONST_NODE_PROVISIONING_MODE_MANUAL = "Manual"
 CONST_NODE_PROVISIONING_MODE_AUTO = "Auto"
+
+# SSH Access Consts
+CONST_SSH_ACCESS_DISABLED = "disbaled"
```

Comment:
Oh, here's a typo
```suggestion
CONST_SSH_ACCESS_DISABLED = "disabled"
```
live test failed https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=87100340&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=a26fe913-e2be-5062-b08e-d15f1acc7ea2&l=695